### ISSUE TITLE
Incomplete .tar.gz for 3.3.0-16

### ISSUE THREAD

**Issue by bvisonl** [pkp/pkp-lib#9671]
## Issue Description
Not 100% sure which section does this issue belongs to, but basically I'm trying to download 3.3.0-16 from [https://pkp.sfu.ca/software/ojs/download/archive/](https://pkp.sfu.ca/software/ojs/download/archive/)and I am noticing that this version is missing a lot of things (such as lib/pkp, the vendor files, etc). 

## How to reproduce / Current behavior
Just download the version and extract it

## What application are you using?
OJS 3.3.0-16

**Comment by jonasraoni** [comment]
@asmecher I just confirmed that all submodules are missing.

**Comment by asmecher** [comment]
Ah, actually, the problem is that the download links are incorrect -- they are going to github in some cases, but need to come from the PKP website where the built packages should be distributed. I'll get those corrected.
